src/platform/export/exportBase.web.ts

```diff
@@ -0,0 +1,185 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+'use strict';
+import * as nbformat from '@jupyterlab/nbformat';
+import { inject, injectable } from 'inversify';
+import { Uri, CancellationToken, NotebookDocument } from 'vscode';
+import * as path from '../../platform/vscode-path/path';
+import { DisplayOptions } from '../../kernels/displayOptions';
+import { executeSilently } from '../../kernels/helpers';
+import { IKernel, IKernelProvider } from '../../kernels/types';
+import { concatMultilineString } from '../../webviews/webview-side/common';
+import { IFileSystem } from '../common/platform/types';
+import { PythonEnvironment } from '../pythonEnvironments/info';
+import { ExportUtilBase } from './exportUtil';
+import { ExportFormat, IExportBase, IExportDialog, INbConvertExport } from './types';
+import { traceLog } from '../logging';
+import { reportAction } from '../progress/decorator';
+import { ReportableAction } from '../progress/types';
+
+@injectable()
+export class ExportBase implements INbConvertExport, IExportBase {
```

Comment:
ExportHelper is probably a better name now since they aren't being extended anymore